content/manager_webui/deployments-page.md

```diff
@@ -14,6 +14,22 @@ relationships_link: dsl-spec-relationships.html
 
 When clicking on the `Deployments` tab and choosing a deployment you will be able to choose one of the following:
 
+## Actions Available On A Deployment
+The table presented on the deployment page contains a dropdown menu allowing to perform actions on a deployment. Available actions are: execute workflow, delete, and update deployment.<br/>
+![Deployment actions]({{< img "ui/deploymentActions/actions-dropdown.png" >}})
+
+### Execute Workflow
+To execute a workflow simply choose the desired workflow name, fill the requested parameters and click `Confirm`.<br/>
+![Execute workflow]({{< img "ui/deploymentActions/execute.png" >}})
+
+### Delete Deployment
+To delete the deployment simply click `Yes`.<br/>
+![Delete deployment]({{< img "ui/deploymentActions/delete-deployment.png" >}})
+
+### Update Deployment
+To Update the deployment simply select or provide a URL to a new archive. If inputs are needed select an inputs file. If the main blueprint file isn't named 'blueprint.yaml' enter the blueprint filename. Lastly you can choose whether to run `Install` and `Uninstall` workflows on the changes  - by default install will run on the added sub-graph and uninstall on the removed subgraph, but the behavior can be overridden by marking which of the workflows should run, or providing a custom workflow of your own. More details on deployment update can be found at <http://docs.getcloudify.org/3.4.0/manager/update-deployment/> <br/>
```

Comment:
If the main blueprint file isn't named 'blueprint.yaml' enter the blueprint filename -> If the main blueprint file isn't named 'blueprint.yaml', type a different name